conductor_api/src/holochain.rs

```diff
@@ -182,9 +182,6 @@ impl Holochain {
 
     /// activate the Holochain instance
     pub fn start(&mut self) -> Result<(), HolochainInstanceError> {
-        if self.active {
-            return Err(HolochainInstanceError::InstanceAlreadyActive);
-        }
```

Comment:
I don't think we should remove this here. Rather handle the error where this is called within `Conductor`...